Autogarage/src/model/SimulatorModel.java

```diff
@@ -71,12 +81,16 @@ public SimulatorModel(SimulatorController controller) {
         paymentCarQueue = new CarQueue();
         exitCarQueue = new CarQueue();
         
-        //Construct the bank.
+        //Construct the banks.
         bank = new Bank();
+        adHocBank = new Bank();

```

Comment:
Het is beter om alles in 1 bank te verwerken denk ik. Bespreek morgen nog wel ff.